SimCore/src/SimCore/BiasOperators/PhotoNuclear.cxx

```diff
@@ -68,9 +68,12 @@ G4VBiasingOperation* PhotoNuclear::ProposeOccurenceBiasingOperation(
 
     double emXsecBiased = std::max(
         emXsecUnbiased + pnXsecUnbiased_ - pnXsecBiased_, pnXsecUnbiased_);
-    if (emXsecBiased == pnXsecUnbiased_) {
-      ldmx_log(debug) << "EM XS = PN unbiased XS! The biasing factor ("
-                      << factor_ << ") is too large";
+    auto material = track->GetMaterial();
+    if ((emXsecBiased == pnXsecUnbiased_) && material->GetName() == "G4_W") {
```

Comment:
Is there a way to do the material check with a pointer instead? I know @EinarElen has found a large number of time consuming string comparisons where a pointer would work just fine and importantly, much faster. (I don't know how often we will in practice fulfill the first condition of this AND – hopefully we find the sweet spot before launching big productions – so it might not be a big deal, but could serve as an example of how to access this type of G4 info.)